recommender/beta/cloud-client/README.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started with Recommender and the Google Java API Client library
+
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=README.md&cloudshell_working_dir=recommender/beta/cloud-client/">
+<img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
+
+[Cloud Recommender](https://cloud.google.com/recommender/) is a service on Google Cloud that provides 
+usage recommendations for Cloud products and services. This sample Java 
+application demonstrates how to access the Recommender API using the 
+[Google Cloud Client Library for Java](https://github.com/GoogleCloudPlatform/google-cloud-java).
+
+## Quickstart
+
+### Setup
+- Install [Maven](http://maven.apache.org/).
```

Comment:
Preferable we can point to the "Setting up a Java environment" page on GCP which is a little more detailed on what the need for their environment.

Reply:
Done, linked to Setting up a Java environment page for the first step.